pages/_app.js

```diff
@@ -10,12 +10,17 @@ export default function App({ Component, pageProps }) {
   function handleAddRecipe(newData) {
     setRecipes([...recipes, newData]);
   }
+
+  function handleDeleteRecipe(id) {
+    setRecipes(recipes.filter((recipe) => recipe.id !== id));
+  }
   return (
     <>
       <GlobalStyle />
       <Component
         {...pageProps}
         handleAddRecipe={handleAddRecipe}
+        handleDeleteRecipe={handleDeleteRecipe}
```

Comment:
```suggestion
        onAddRecipe={handleAddRecipe}
        onDeleteRecipe={handleDeleteRecipe}
```

to follow common lifting up state naming convention